src/components/vertical-divider/vertical-divider.component.tsx

```diff
@@ -101,13 +104,37 @@ type TintRange =
   | 98
   | 99
   | 100;
+
 export interface VerticalDividerPropTypes extends SpaceProps {
   h?: number | string;
+  height?: number | string;
   displayInline?: boolean;
   /** the supported rage is 1-100  */
   tint?: TintRange;
 }
 
-declare function VerticalDivider(props: VerticalDividerPropTypes): JSX.Element;
+const VerticalDivider = ({
+  h,
+  height,
+  displayInline = false,
+  tint = 80,
+  ...props
+}: VerticalDividerPropTypes): JSX.Element => (
+  <MenuContext.Consumer>
```

Comment:
Let's consume the context in the functional component way using the `useContext` hook instead, it makes the jsx much cleaner